### PR TITLE
More changes for Panel, Icon and jBox

### PR DESCRIPTION
**Pull request by mbaas2**

### CODE REVIEW

File: Config/Resources.xml [hunk 1]
```diff
@@ -13,7 +13,10 @@
 		<name>dcAccordion</name>
 		<style>/Dyalog/dcAccordion/dcAccordion.css</style>
 	</resource>
-
+	<resource>
+		<name>dcPanel</name>
+		<style>/Dyalog/Panel/Panel.css</style>
```
Comment (bpbecker) [inline Config/Resources.xml]: This should be dcPanel.css to be consistent with other Dyalog controls.

File: HTML/_DC/Icon.dyalog [hunk 2]
```diff
@@ -38,11 +38,11 @@
       :EndIf
     ∇
 
-    ∇ r←Render;prefix;spec;icon;classes;n
+    ∇ r←Render;prefix;spec;icon;classes;n;Content∆
       :Access public
       Spec←eis Spec
       Spec,←eis Content
-      Content←⍬
+      Content∆←⍬
```
Comment (bpbecker) [inline HTML/_DC/Icon.dyalog]: Please explain the rationale behind Content∆

File: HTML/_JS/jBox.dyalog [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿:Class jBox : #._JQ._jqWidget
 ⍝ Description:: jBox widget
 ⍝
-⍝ Constructor:: [type [message [content]]]
+⍝ Constructor:: [Type [Message [Content]]]
```
Comment (bpbecker) [inline HTML/_JS/jBox.dyalog]: Our convention is to use lower case in constructor arguments and upper case for public Fields

File: HTML/_JS/jBox.dyalog [hunk 4]
```diff
@@ -52,33 +53,36 @@
       'Invalid jBox Type'⎕SIGNAL(ind>⍴Types)/11
       Type←ind⊃Types
       Container.Content←Content
-      'content'Set renderIt Message
+      'content'Set renderIt New _.span Message   ⍝ seems to be neccessary to render EVERYTHING!
```
Comment (bpbecker) [inline HTML/_JS/jBox.dyalog]: I've incorporated this into my copy

**Comment by bpbecker** [comment]
Michael, 
In the future, can you please use separate pull requests when updating multiple things?
I'm closing this request without merging because it conflicts with some changes that were already present.
I've gone ahead and manually merged some of the changes you've suggested.
Also, jBoxAdvanced is doing way too much for a single example page - we should split it up.

**Comment by mbaas2** [comment]
Ok, I'll split up Advanced. I was afraid I would create conflicts when
pushing repeatedly while prior changes were not accepted - which is why I
tried to put more stuff into single "pushes". I'll change that and get more
courageous ;-)

2016-09-18 23:27 GMT+02:00 Brian Becker <notifications@github.com>:

**Pull request closed without merging**